changelog/fix18397.dd

```diff
@@ -1,18 +1,18 @@
-Changed std.conv.hexString() to return an immutable string literal
-rather than an array of ubytes. This is more in keeping with
-the documentation that says it is a replacement for the deprecated
-x"deadbeef" string literal syntax.
+Changed `std.conv.hexString` to return an immutable string literal
 
-https://dlang.org/phobos/std_conv.html#hexString
+$(REF hexString, std,conv) now returns an immutable string literal rather than an array of ubytes.
+This is more in keeping with
+the documentation that says it is a replacement for the deprecated
+`x"deadbeef"` string literal syntax.
 
 The benefits of this change are:
 
-* behavior consistency with the documentation
-* the hexString template instantiations no longer appear in the
-generated object file
-* the generated object file no longer contains references to TypeInfo
-and various druntime functions
-* it is now compatible with -betterC mode
+$(UL
+  $(LI behavior consistency with the documentation)
+  $(LI the `hexString` template instantiations no longer appear in the generated object file)
+  $(LI the generated object file no longer contains references to `TypeInfo` and various druntime functions)
+  $(LI it is now compatible with `-betterC` mode)
+)
```

Comment:
FYI @WalterBright even if `changed.d` overlays some goodies over Ddoc (no need for the ugly `$(P)`, Ddoc is still used in the background :/